app/src/main/AndroidManifest.xml

```diff
@@ -25,7 +25,8 @@
       android:theme="@style/OppiaThemeWithoutActionBar" />
     <activity
       android:name=".app.completedstorylist.CompletedStoryListActivity"
-      android:theme="@style/OppiaThemeWithoutActionBar" />
+      android:theme="@style/OppiaThemeWithoutActionBar"
+      android:label="@string/completed_story_list_activity_title"/>
```

Comment:
Press `Control + Alt + L` - reformatting shortcut to fix the formatting.

There should be space before `/>`

Reply:
Done